src/manager.js

```diff
@@ -54,6 +57,14 @@ AdManager.prototype.bindContext = function() {
   this.onSlotOnload = this.onSlotOnload.bind(this);
 };
 
+AdManager.prototype.prebidInit = function() {
+  this.pbjs = window.pbjs;
+  if (this.pbjs && this.options.pbjsEnabled && this.adUnits.pbjsSizeConfig) {
+    this.pbjs.cmd.push(() => {
```

Comment:
This looks like an ES6 function, remember we're not currently transpiling so this needs to be converted to ES5